apps/user-profile/src/assets/config/defaultConfig.json

```diff
@@ -73,6 +73,7 @@
   "mfa": {
     "api_url": "https://id.muni.cz/mfaapi/",
     "enable_security_image": true,
+    "enable_detail_settings": false,
```

Comment:
Please update the config doc:
https://perunaai.atlassian.net/wiki/spaces/STRIBOG/pages/3440714/Configuration+of+new+GUI+applications

Reply:
Done :+1: